apps/cookbook/src/app/showcase/grid-showcase/grid-showcase.component.html

```diff
@@ -5,15 +5,11 @@
     adapt the layout to different screen sizes.
   </p>
   <h3>Breakpoints</h3>
-  <p>
-    Each breakpoint is named using t-shirt sizes and relates to a defined screen size or bigger:
-  </p>
+  <p>Each breakpoint is named using t-shirt sizes and maps to a defined screen size or bigger:</p>
   <ul>
-    <li>xs, extra-small: 0px</li>
-    <li>sm, small: 632px</li>
-    <li>md, medium: 768px</li>
-    <li>lg, large: 992px</li>
-    <li>xl, extra-large: 1440px</li>
+    <li *ngFor="let breakpoint of breakpoints | keyvalue: originalOrder">
+      {{ breakpointKeyToShorthand.get(breakpoint.key) }}: {{ breakpoint.value }}
```

Comment:
Do we need a map or could we simply use a typed object literal? 🤔 
```suggestion
      {{ breakpointToShorthandMap{breakpoint.key} }}: {{ breakpoint.value }}
```